src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,27 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int MAX_COUNT_CHAR = 5;
+    private int character;
+    private int currentCharacter = 1;
```

Comment:
do we need this field or it can be just a local variable?

Reply:
why = 1?

Reply:
> do we need this field or it can be just a local variable?
```
 for (Character value : letters) {
                if (currentCharacter > MAX_COUNT_CHAR) {
                    break;
                }
                result.append(value);
                currentCharacter++;
            }
```
в цьому циклі спочатку додається елемент ( тобто вже currentCharacter == 1) і потім вмикається лічильник.
Якщо ця змінна буде локальною, є ризик, що їй присвоять значення "0".  І тоді,  MAX_COUNT_CHAR треба буде зменшити на один, щоб додавалось 5 елементів, а не шість. Програма буде не коректно виконуватися.
Чи краще переписати цей шматок?))

Reply:
я думаю це просто переписати щоб починалося з 0 і тобі не потрібно ця змінна як поле

Reply:
переписав та подивився як виглядає - Ти абсолютно правий
нема потреби придумувати непотрібні змінні((